src/reactor.h

```diff
@@ -191,12 +200,14 @@ class Reactor : public cyclus::Facility,
   ///////// cycle params ///////////
   #pragma cyclus var { \
     "doc": "The duration of a full operational cycle (excluding refueling time) in time steps.", \
+    "uilabel": "Cycle Length", \
     "units": "time steps", \
   }
   int cycle_time;
   #pragma cyclus var { \
     "doc": "The duration of a full refueling period - the minimum time between" \
            " a cycle end and the start of the next cycle.", \
+    "uilabel": "Refueling Outage Length", \
```

Comment:
I slightly prefer  "Refueling Outage Duration".  For some reason "length" is unintuitive to me here (but not in the 'Cycle Length' example).